webrecorder/browsertrix
# fix: Update browser title

Resolves https://github.com/webrecorder/browsertrix/issues/1983



### Changes

Updates browser title when visiting the following pages:

- Superadmin dashboard
- Org top-level pages
- Account settings

Follows a similar approach to [React <title>](https://react.dev/reference/react-dom/components/title#special-rendering-behavior).

### Manual testing

1. Log in
2. Navigate around org pages. Verify browser title bar updates to reflect the top-level org navigation.

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Brower Profiles | <img width="829" alt="Screenshot 2024-08-28 at 1 32 27 PM" src="https://github.com/user-attachments/assets/30903668-c3f9-4c95-86a8-178296baabf0"> |


### Follow-ups

Once file refactors in https://github.com/webrecorder/browsertrix/pull/2053 and https://github.com/webrecorder/browsertrix/pull/2020 are merged we can add page titles for detail views as well.
